Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -873,6 +889,9 @@ def early_shutdown(self):
             yield self.ipv8.unload_overlay(self.tunnel_community)
             yield self.ipv8.unload_overlay(self.triblerchain_community)
 
+        if self.popular_community:
```

Comment:
Why do you unload this separately? The tunnel and triblerchain are only unloaded the way they are to enforce the unload ordering.